vmhost/hostCore/host.go

```diff
@@ -29,6 +30,22 @@ var _ vmhost.VMHost = (*vmHost)(nil)
 const minExecutionTimeout = time.Second
 const internalVMErrors = "internalVMErrors"
 
+// allFlags must have all flags used by mx-chain-vm-v1_4-go in the current version
+var allFlags = []core.EnableEpochFlag{
```

Comment:
I think it will be better to keep the all flags definitions together with the flags. Reason: when we will add a new flag, it will be harder to forget to add it in the list

Reply:
I agree.. the reason why on this repo I separated them is to avoid cyclic imports.. added a comment to avoid the mentioned situation. The second option would have been an exported AllFlags slice, which may cause confusions.